lib/board.rb

```diff
@@ -0,0 +1,52 @@
+
+class Board
+  attr_accessor :cells
+    def initialize(cells = [" ", " "," "," "," "," "," "," "," "])
+      @cells = cells
+    end
+
+    def reset!
+      @cells = [" ", " "," "," "," "," "," "," "," "]
+    end
+    def display
+      puts " #{@cells[0]} | #{@cells[1]} | #{@cells[2]} "
+      puts "-----------"
+      puts " #{@cells[3]} | #{@cells[4]} | #{@cells[5]} "
+      puts "-----------"
+      puts " #{@cells[6]} | #{@cells[7]} | #{@cells[8]} "
+    end
+
+    def position(normalized_index)
+      @cells[normalized_index.to_i - 1]
+    end
+    def taken?(normalized_index)
+      cell = position(normalized_index)
+      return false if(cell != "X" && cell != "O")
+      return true
+    end
+
+    def valid_move?(normalized_index)
```

Comment:
Use 'and' operator (`&&`) to combine multiple logical expressions and utilize their implicit return value.
```
normalized_index.between?(0,8) && !taken(normalized_index)
```